Firestore/Source/Remote/FSTDatastore.m

```diff
@@ -436,10 +438,56 @@ + (void)prepareHeadersForRPC:(GRPCCall *)rpc
 
 @end
 
+#pragma mark - GRXFilter
+
+/** Filter class that allows disabling of GRPC callbacks. */
+@interface GRXFilter : NSObject <GRXWriteable>
+
+- (instancetype)initWithStream:(FSTStream *)stream NS_DESIGNATED_INITIALIZER;
+- (instancetype)init NS_UNAVAILABLE;
+
+@property(atomic, readwrite, getter=isPassthrough) BOOL passthrough;
```

Comment:
The class comment discusses "disabling" callbacks but this property is about passthrough. It seems like the concepts should align. Also "passthrough" is sort of vague when read at the call site. A better name would more directly convey what this thing is doing. Consider something like "callbacksEnabled" instead.

Also, since this such a core component of what this thing is responsible for, maybe make this just a method: `suppressCallbacks` or `disableCallbacks`.

Reply:
That works for me (went with the suggested names)/